xeda/plugins/lwc/flows/timing_and_verification.py

```diff
@@ -352,4 +353,11 @@ def parse_reports(self):
 
             results['TV:' + rc_name] = rc_results
 
-        results['success'] = True
+        
+        if results['success'] != False:
```

Comment:
Why is this required?

Reply:
I removed the return on line 340,  so `results['success']` may be set to false from a run. We do not want to overwrite that result, so we only set true if its not false.